steps/hearing/options/options.js

```diff
@@ -0,0 +1,7 @@
+const options = {
+  telephone: 'telephone',
+  video: 'video',
```

Comment:
We have welsh language as well, please use this hard coded value into content.en.json file